plugin/util/parse_bindings.go

```diff
@@ -88,12 +88,15 @@ func parseBindingObjList(topList *ast.ObjectList) (map[string]StringSet, error)
 
 		resourceList := item.Val.(*ast.ObjectType).List
 		for _, rolesItem := range resourceList.Items {
-			key := rolesItem.Keys[0].Token.Text
-			switch key {
+			if len(rolesItem.Keys) == 0 || rolesItem.Keys[0].Token.Value() == nil {
```

Comment:
I think we also need to check if `rolesItem.Keys[0].Token` isn't nil before calling `.Value()`

Reply:
This seems to be the only issue left.